src/styles/utils/_mixins.scss

```diff
@@ -55,5 +55,12 @@
   @include on-desktop {
     --columns: 24;
   }
+
+  @include on-tablet {
+    padding-inline: 24px;
+  }
```

Comment:
Add more to adapt your menu.
Because only the bottom buttons stretch, but also it should have min-with and after 639px menu should disappear.